plugins/woocommerce/client/legacy/js/frontend/single-product.js

```diff
@@ -99,7 +99,7 @@ jQuery( function( $ ) {
 		} )
 		// Star ratings for comments
 		.on( 'init', '#rating', function() {
-			$( '#rating' )
+			$( this )
```

Comment:
This fixes a long standing bug when multiple review blocks are used within the same template, cause multiple star ratings for the rating field first rendered: https://github.com/woocommerce/woocommerce/issues/51373
This fixes it, instead of using `#rating` which targets the first one everytime, we use `this` to actually target the star rating in the correct context.

Reply:
I will extract this into its own PR for better history. I don't feel good mixing experimental and production features together.